src/i_rumble.c

```diff
@@ -297,16 +302,15 @@ static void InitFFT(int rate, int step)
     last_rate = rate;
     FreeFFT();
 
-    // FFT size must be a power of two (see pffft.h).
+    // Set FFT size to a power of two for speed.
     const uint32_t step2 = RoundUpPowerOfTwo(step);
     fft.size = BETWEEN(128, 8192, step2);
 
-    fft.setup = pffft_new_setup(fft.size, PFFFT_REAL);
+    fft.in = fftwf_malloc(sizeof(*fft.in) * fft.size);
+    fft.window = fftwf_malloc(sizeof(*fft.window) * fft.size);
+    fft.out = fftwf_malloc(sizeof(*fft.out) * (fft.size / 2 + 1));
 
-    const size_t size = sizeof(float) * fft.size;
-    fft.in = pffft_aligned_malloc(size);
-    fft.out = pffft_aligned_malloc(size);
-    fft.window = pffft_aligned_malloc(size);
+    fft.plan = fftwf_plan_dft_r2c_1d(fft.size, fft.in, fft.out, FFTW_ESTIMATE);
```

Comment:
Have you tried different flags, like e.g. `FFTW_MEASURE` or `FFTW_DESTROY_INPUT` against `FFTW_PRESERVE_INPUT`?

Reply:
`FFTW_MEASURE` is much slower and the others don't apply to our use case. I also tried the in-place transforms method described [here](https://fftw.org/doc/One_002dDimensional-DFTs-of-Real-Data.html#One_002dDimensional-DFTs-of-Real-Data) and it was slower than out-of-place transforms (out-of-place transforms is shown in the above code).

Reply:
Just to be sure, I'm going to benchmark a few other settings this weekend.

Reply:
I tried a number of different methods (c2c, r2c, various plan flags) and pffft is still the fastest, so this PR is good to go.